views/partials/layout.njk

```diff
@@ -48,7 +48,7 @@
 {# Run JavaScript at end of the
 <body>, to avoid blocking the initial render. #}
 <script src="/assets/govuk/all.js"></script>
-<script src="/static/js/main.js"></script>
```

Comment:
must be some magic going on so that these paths work in dev - both https://whereabouts-dev.prison.service.justice.gov.uk/js/main.js and https://whereabouts-dev.prison.service.justice.gov.uk/static/js/main.js serve the same content.  However after my changes the `static` version requires the user to be logged in :shrug: